src/tests/class_tests/openms/source/BinnedSumAgreeingIntensities_test.cpp

```diff
@@ -93,7 +93,10 @@ START_SECTION((double operator()(const BinnedSpectrum &spec1, const BinnedSpectr
   BinnedSpectrum bs2 (1.5,2,s2);
 
   double score = (*ptr)(bs1, bs2);
-  TEST_REAL_SIMILAR(score, 0.997576)
+  TEST_REAL_SIMILAR(score, 0.99707)
```

Comment:
any reason why the numerics change?

Reply:
the reason is that the score normalization is performed by dividing by (sum1 + sum2).
Previously, sum1 and sum2 were calculated only on a subset of the dimensions (the minimum of the highest non-zero indices of both spectra). It is now calculated for the full spectrum.
I looked into the original publication and, as I understood it, the way I am doing it now is correct.

Reply:
in other words: if s1 had the last non-zero coefficient at, say, index 100 and s2 at index 200, then the sums were previously only calculated up to index 100. Now it is calculated for the full range of each spectrum.